src/main/java/io/jenkins/plugins/eggplant/EggplantRunnerBuilder.java

```diff
@@ -372,15 +365,23 @@ private Boolean isValidDecimal(String value){
                 return false;
         }
 
-        private Boolean isValidLogLevel(String value){
-            Pattern p = Pattern.compile("INFO|DEBUG|WARNING|ERROR");
-            Boolean isMatch=p.matcher(value).matches();
-            if(isMatch)
-                return true;
-            else
-                return false;
-        }
-
     }
 
+    public enum LogLevel {
```

Comment:
public enum LogLevel() can move to a new folder under java/io/jenkins/plugins/eggplant/**common**
Move the enum into new java class file in the common folder.
File name: LogLevel

Reply:
Ok got it.